dirsrvtests/tests/suites/clu/dsidm_group_test.py

```diff
@@ -0,0 +1,416 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2024 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+
+import pytest
+import logging
+import os
+
+from lib389 import DEFAULT_SUFFIX
+from lib389.cli_idm.group import list, get, get_dn, create, delete, modify, rename
+from lib389.topologies import topology_st
+from lib389.cli_base import FakeArgs
+from lib389.utils import ds_is_older, ensure_str
+from lib389.idm.group import Groups
+from . import check_value_in_log_and_reset
+
+pytestmark = pytest.mark.tier0
+
+logging.getLogger(__name__).setLevel(logging.DEBUG)
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def create_test_group(topology_st, request):
+    group_name = "test_group_2000"
+    groups = Groups(topology_st.standalone, DEFAULT_SUFFIX)
+
+    log.info('Create test group')
+    if groups.exists(group_name):
+        test_group = groups.get(group_name)
+        test_group.delete()
+    else:
+        test_group = groups.create_test_group()
+
+    def fin():
+        log.info('Delete test group')
+        if test_group.exists():
+            test_group.delete()
+
+    request.addfinalizer(fin)
+
+
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_group_create(topology_st):
```

Comment:
I wonder if you should not use the create_test_group fixture then assert that the group exists

Reply:
Thank you for review. I left the creation test as it was, since it also checks that the correct messages are generated, which is not covered by the fixture. It seems redundant to include it to the fixture, as it would perform the check every time it's used, therefore in every single test.